rspirv/binary/disassemble.rs

```diff
@@ -197,10 +197,10 @@ fn disas_constant(inst: &dr::Instruction, type_tracker: &tracker::TypeTracker) -
     debug_assert_eq!(inst.operands.len(), 1);
     let literal_type = type_tracker.resolve(inst.result_type.unwrap());
     match inst.operands[0] {
-        LiteralInt32(value) => disas_instruction(inst, " ", |_| {
```

Comment:
the operand won't be disassembled correctly in case of a floating point literal type due to the underlying `disas_literal_int` code which does some type checking. Should be extended to support `Float` types as well otherwise we print an empty string.

Reply:
ah, my mistake. nice catch!

Reply:
hmm, i looked into it and it looks like this might require something similar to [this module in `spirv-tools`](https://github.com/KhronosGroup/SPIRV-Tools/blob/4dbc66380dd63aabbd33c38198008449d0a5807a/source/util/hex_float.h) in order to properly deal with NaNs, infinity, 16-bit floats, etc. i think this is outside the scope of this PR, though, so I left a TODO comment and decided to just rely on the `Display` impls of `f32` and `f64` to handle most usages, as i believe this is what the behavior was before anyway.